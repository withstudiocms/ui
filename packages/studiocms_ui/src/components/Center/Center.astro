---
import './center.css';
---
<!-- This was made for Jumper, to keep him happy <3 -->
<div class="sui-center">
  <slot />
</div>
