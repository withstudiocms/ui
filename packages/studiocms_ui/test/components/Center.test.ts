import { describe, expect } from 'vitest';
import Center from '../../src/components/Center/Center.astro';
import { test } from '../fixtures/vitest/AstroContainer';

describe('Center Component', () => {
	test('renders Center component correctly', async ({ renderComponent }) => {
		const result = await renderComponent(Center, 'Center', {
			slots: { default: 'Centered Content' },
		});
		expect(result).toMatchSnapshot();
	});
});
