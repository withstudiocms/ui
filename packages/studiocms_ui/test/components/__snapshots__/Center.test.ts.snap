// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Center Component > renders Center component correctly 1`] = `"<div class="sui-center"> Centered Content </div>"`;
